cpp/src/parquet/column_writer-test.cc

```diff
@@ -40,6 +40,23 @@ using schema::PrimitiveNode;
 
 namespace test {
 
+namespace {
+const int32_t kGZipDefaultCompressionLevel =
+    GetCompressionCodecDefaultCompressionLevel(Compression::GZIP);
+const int32_t kZSTDDefaultCompressionLevel =
+    GetCompressionCodecDefaultCompressionLevel(Compression::ZSTD);
+const int32_t kBrotliDDefaultCompressionLevel =
+    GetCompressionCodecDefaultCompressionLevel(Compression::BROTLI);
+const int32_t kSnappyDDefaultCompressionLevel =
+    GetCompressionCodecDefaultCompressionLevel(Compression::SNAPPY);
+const int32_t kLZODDefaultCompressionLevel =
+    GetCompressionCodecDefaultCompressionLevel(Compression::LZO);
+const int32_t kLZ4DDefaultCompressionLevel =
+    GetCompressionCodecDefaultCompressionLevel(Compression::LZ4);
+const int32_t kUncompressedDefaultCompressionLevel =
```

Comment:
What's the point of all this? It seems this is gratuitously complicating the code, no?

Reply:
The point was to avoid having to write GetCompressionCodecDefaultCompressionLevel(...) in many places.
You might argue that than we should set a default value for the compression level parameter. However, I prefer it to have to be explicitly set within Arrow/Parquet while making it optional for the user. The way I see it this avoids potential improper calls to functions which have multiple default parameters, AND avoids having many functions be duplicated whenever it's not possible to have a default value.
Anyway, I will remove these constants.

Reply:
My point is that the default values are known in Arrow somewhere and there's no point in duplicating them in Parquet. Let Arrow fill in the default if no specific value is passed.